projects/ngx-formentry/src/components/ngx-remote-select/remote-select.component.html

```diff
@@ -13,6 +13,7 @@
     typeToSearchText="Please enter 2 or more characters"
     [typeahead]="remoteOptionInput$"
     [(ngModel)]="selectedRemoteOptions"
+    [appendTo]="'body'"
```

Comment:
Yah, that's what the docs had, I will test with double quotes only and see. My thinking was that we are appending a string body to the select.

Reply:
I don't think `body` is the right selector for this. It works in the self-contained app here, but that's because the app only displays the form. In the form-engine context, however, this is almost certainly wrong. In the context of the form engine on dev3, it probably makes more sense to have this as `appendTo="my-app-fe-wrapper form"`, i.e., to the form element itself.